data-prepper-core/src/test/resources/valid_data_prepper_config_with_processor_shutdown_timeout.yml

```diff
@@ -0,0 +1,3 @@
+serverPort: 5678
+ssl: false
+processorShutdownTimeout: 1s
```

Comment:
It could be good to have a valid configuration which uses an ISO-8601 duration in addition to these two with the `1s` value.